BattleNetwork/bnTile.h

```diff
@@ -56,7 +56,7 @@ namespace Battle {
       solid = 2,
     };
 
-    friend class Field;
+    friend Field;
```

Comment:
this is weird. what errors were related? Never seen a class friended without the keyword in front.